### PR TITLE
Crud Build Out

### PR DESCRIPTION
**Pull request by michaelrtobias**

### CODE REVIEW

**Comment by michaelrtobias** [comment]
@chanceknguyen

File: README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  - Uses query parameters to find games based on 'id'.
+    - 'id' is not the object _id, but an id given in the original data generation script
+
+- Update | PUT: '/moist-air/game'
+  - Updates one game
+  - Uses query parameters to find games based on 'id'.
+    - 'id' is not the object _id, but an id given in the original data generation script
+  - Currently updates the price of the game. Can be expanded to many other peramteres for update
+
+- Delete | DELETE: '/moist-air/game'
```
Comment (chanceknguyen) [inline README.md:46]: Nice overview, are these all of the specific query parameters needed or are there more specifically that should be noted?

File: server/index.js [hunk 2]
```diff
@@ -7,9 +7,20 @@ const port = 3000;
 const Game = require('../db/index.js');
 
 app.use(cors())
-app.use(express.json())
+app.use(express.json());
+app.use(bodyParser.urlencoded({extended: false}));
```
Comment (chanceknguyen) [inline server/index.js:11]: Any particular reason why you are using bodyParser middleware as well as express' native body parsing middleware with express.json?

**Review by chanceknguyen — APPROVED** [review]
Overall this looks good to me! One feedback that Chris had for me (that I have also not incorporated yet) was to consider removing the db actions from the server and add a middle file that handles it and gets it ready for the database. For example, the server should literally only receive data, pass it to the middle file, which then parses the data and gets it suitable for the db functions, then the db does it thing and send back the response data do the server. The reasoning behind this is to make the API server database agnostic and allow for quick swaps between databases.

File: server/index.js (hunk 2 above)
Comment (michaelrtobias) [reply server/index.js:11]: I meant to delete body parser. Its gone now.

File: README.md (hunk 1 above)
Comment (michaelrtobias) [reply README.md:46]: Those are the only query parameters. Just the id

**Comment by michaelrtobias** [comment]
That makes a lot of sense. Will be factoring that in tomorrow. Thanks!

> Overall this looks good to me! One feedback that Chris had for me (that I have also not incorporated yet) was to consider removing the db actions from the server and add a middle file that handles it and gets it ready for the database. For example, the server should literally only receive data, pass it to the middle file, which then parses the data and gets it suitable for the db functions, then the db does it thing and send back the response data do the server. The reasoning behind this is to make the API server database agnostic and allow for quick swaps between databases.

**Pull request merged**